charts/kube-prometheus-stack/values.yaml

```diff
@@ -793,6 +793,7 @@ grafana:
     datasources:
       enabled: true
       defaultDatasourceEnabled: true
+      IsDefaultDatasource: true
```

Comment:
I'm totally open for a variable name suggestions

Reply:
I would probably suggest isDefault here